src/pages/index.tsx

```diff
@@ -46,17 +56,35 @@ const Home: NextPage = () => {
 				<Link href="/leeszaal/leeszaal-8">Ga naar leeszaal</Link>
 			</div>
 
-			<div className="l-container u-mb-64">
-				<ReadingRoomCardList items={readingRooms} limit={!areAllReadingRoomsVisible} />
-			</div>
+			<div className="l-container u-mb-80">
```

Comment:
```suggestion
			<div className="l-container u-mb-50 u-mb-80:md">
```

Reply:
Fixed, heb wel 48 ipv 50 gepakt om in de meervouden van 8 te blijven werken.